UpKeep/Models/InventoryItemModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UpKeepData.Entity;
+
+namespace UpKeep.Models
+{
+    public class InventoryItemModel
+    {
+        public InventoryItemModel()
+        {
+            InventoryItemMaintenanceTasks = new HashSet<InventoryItemMaintenanceTask>();
+        }
+
+        public int InventoryItemId { get; set; }
+        public int InventoryItemTypeId { get; set; }
+        public DateTime? PurchaseDate { get; set; }
+        public string? InventoryItemCost { get; set; }
+        public int RoomId { get; set; }
+        public string? QrcodeId { get; set; }
+
+        public virtual InventoryItemType InventoryItemType { get; set; } = null!;
+        public virtual Room Room { get; set; } = null!;
+        public virtual ICollection<InventoryItemMaintenanceTask> InventoryItemMaintenanceTasks { get; set; }
+    }
+}
```

Comment:
might need to add more fields that we can discuss but a good start that we can add onto 💯